packages/notebook/src/widget.ts

```diff
@@ -2387,6 +2401,20 @@ export class Notebook extends StaticNotebook {
     }
   }
 
+  /**
+   * Handle UI change from other collaborators
+   */
+  private _onUiChanged(sharedModel: YNotebook, change: NotebookUIChange) {
+    // Force edit mode for the markdown cells on which other collaborators are working
```

Comment:
Wondering if we should have a finer solution, where we have per-cell awareness on the `rendered` state.

Reply:
Yes, it is probably better to have a per-cell awareness.
And regarding the new signal, I think would be better to not add a new signal but instead add a new property stateChanged to the existing signal. I just changed my mind because I saw that the notebook is using stateChanged to notify that the mode changed https://github.com/jupyterlab/jupyterlab/blob/bd07b0779760e63c2fc147a5544db78e8b9a7282/packages/notebook/src/widget.ts#L1147. I think we should follow this pattern on the shared cell model.

Reply:
Ok :) Let's do that. Thanks for the review!